db/migrations/20230227122704-ss58-prefix.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var keyring = require('@polkadot/keyring');
+var Promise;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+    dbm = options.dbmigrate;
+    type = dbm.dataType;
+    seed = seedLink;
+    Promise = options.Promise;
+};
+
+class Logger {
+    constructor(tableName) {
+        this.tableName = tableName;
+        this.totalRows = 0;
+        this.currentRow = 0;
+    }
+
+    setTotalRows = (totalRows) => {
+        this.totalRows = totalRows;
+    }
+
+    log = () => {
+        this.currentRow++;
+        const {currentRow, totalRows, tableName} = this;
+
+        if (currentRow === totalRows) {
+            console.log(`✅ Done for ${tableName} table`);
+        } else {
+            console.log(`⏳ Migrated ${Math.trunc(currentRow / totalRows * 10000) / 100}% for ${tableName} table`);
+        }
+    }
+}
+
+// Migration script for Account Table
+const convertAccountTable = (db, ss58Format) => {
+    const logger = new Logger('account');
+
+    db.runSql('SELECT account_id, balances, identity from account;', (_, result) => {
+        logger.setTotalRows(result.rows.length);
+        result.rows.forEach(({account_id, balances, identity}) => {
+
+            const nextAddress = keyring.encodeAddress(
+                keyring.decodeAddress(account_id),
+                ss58Format);
+
+            const nextBalances = balances.replace(account_id, nextAddress);
+
+            let queryString = `UPDATE account SET account_id='${nextAddress}', balances='${nextBalances}'`
+
+            if (identity && identity.includes("parent")) {
```

Comment:
I guess this can be replaced with `identity?.includes("parent")`

Reply:
Yes, it can. Thank you!